docs/gallery/advanced_io/streaming.py

```diff
@@ -23,113 +23,111 @@
-for a full updated list of supported store formats.
-
-Streaming Method 2: ROS3
-------------------------
-ROS3 is one of the supported methods for reading data from a remote store. ROS3 stands for "read only S3" and is a
-driver created by the HDF5 Group that allows HDF5 to read HDF5 files stored remotely in s3 buckets. Using this method
-requires that your HDF5 library is installed with the ROS3 driver enabled. This is not the default configuration,
-so you will need to make sure you install the right version of ``h5py`` that has this advanced configuration enabled.
-You can install HDF5 with the ROS3 driver from `conda-forge <https://conda-forge.org/>`_ using ``conda``. You may
-first need to uninstall a currently installed version of ``h5py``.
-
-.. code-block:: bash
-
-   pip uninstall h5py
-   conda install -c conda-forge "h5py>=3.2"
-
-Now instantiate a :py:class:`~pynwb.NWBHDF5IO` object with the S3 URL and specify the driver as "ros3". This
-will download metadata about the file from the S3 bucket to memory. The values of datasets are accessed lazily,
-just like when reading an NWB file stored locally. So, slicing into a dataset will require additional time to
-download the sliced data (and only the sliced data) to memory.
-
-.. code-block:: python
-
-   from pynwb import NWBHDF5IO
-
-   with NWBHDF5IO(s3_url, mode='r', load_namespaces=True, driver='ros3') as io:
-       nwbfile = io.read()
-       print(nwbfile)
-       print(nwbfile.acquisition['lick_times'].time_series['lick_left_times'].data[:])
-
-Which streaming method to choose?
----------------------------------
-
-fsspec has many advantages over ros3:
-
-1. fsspec is easier to install
-2. fsspec supports caching, which will dramatically speed up repeated requests for the
-   same region of data
-3. fsspec automatically retries when s3 fails to return.
-4. fsspec works with other storage backends and
-5. fsspec works with other types of files.
-6. In our hands, fsspec is faster out-of-the-box.
-
-For these reasons, we would recommend use fsspec for most Python users.
 """
 
 # sphinx_gallery_thumbnail_path = 'figures/gallery_thumbnails_streaming.png'
+
+from dandi.dandiapi import DandiAPIClient
+
+dandiset_id = '000006'  # ephys dataset from the Svoboda Lab
+filepath = 'sub-anm372795/sub-anm372795_ses-20170718.nwb'  # 450 kB file
+with DandiAPIClient() as client:
+    asset = client.get_dandiset(dandiset_id, 'draft').get_asset_by_path(filepath)
+    s3_url = asset.get_content_url(follow_redirects=1, strip_query=True)
+
+##############################################
+# Streaming Method 1: fsspec
+# --------------------------
+# fsspec is another data streaming approach that is quite flexible and has several performance advantages. This library
+# creates a virtual filesystem for remote stores. With this approach, a virtual file is created for the file and
```

Comment:
```suggestion
# The ffspec library provides a flexible and easy-to-use mechanism for creating a virtual filesystem for 
# remote stores and streaming remote NWB files.  With this approach, a virtual file is created for the file and
```

The main reason for the suggestion is that I found the "is another data streaming approach" confusing, since this is the first approach that is being listed.